CommonCompilerConfig.cmake

```diff
@@ -253,40 +255,16 @@ else()
 #        endif()
 
 
-	#
-	# Determine stdlib settings
-	#
-	set(stdlib_cxx)
-	set(stdlib_common)
-
-	if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-	  set(stdlib_cxx ${stdlib_cxx} -stdlib=libc++)
-	endif()
-
-# TODO: investigate if we should use static or shared stdlib and gcc lib.
```

Comment:
`-static` never worked properly with gcc, so we'll use musl for the binary releases.